app/controllers/reviews_controller.rb

```diff
@@ -1,17 +1,27 @@
 class ReviewsController < ApplicationController
-  skip_before_action :require_login, :only => [:create, :show]
   skip_before_action :find_order
+  skip_before_action :require_login, :only => [:create]
   
   def create
     @review = Review.new(review_params)
+    current_user = User.find_by(id: session[:user_id])
```

Comment:
```suggestion
    current_user = current_user
```

(I think this should call the current_user method in app controller)